sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/parquet/ParquetScanBuilder.scala

```diff
@@ -53,16 +53,14 @@ case class ParquetScanBuilder(
   override protected val supportsNestedSchemaPruning: Boolean = true
 
   override def pushDataFilters(dataFilters: Array[Filter]): Array[Filter] = {
-    val sqlConf = sparkSession.sessionState.conf
-    if (sqlConf.parquetFilterPushDown) {
```

Comment:
The original code looks okay to me. Could you confirm that this part also has a broken type annotations before this PR? Or, is this simply revised to reuse `conf` instead of `sqlConf` at line 56?